Slam/ShowQueueViewController.swift

```diff
@@ -1,20 +1,41 @@
 import UIKit
 
 public class ShowQueueViewController: UIViewController {
-
-    @IBOutlet weak var currentQueue: UITableView!
+    @IBOutlet weak public var currentQueue: UITableView!
 
     public var queueManager: QueueManager?
-    public var queue: Queue?
+    public var queue: QueueView?
 
-    public override func viewDidLoad() {
+    override public func viewDidLoad() {
         super.viewDidLoad()
-        queue = Queue(display: currentQueue)
-        queueManager = QueueManager(queue: queue!)
+        initializeDelegates()
+        if (!self.isRunningTests()) {
+            refreshQueue()
+        }
```

Comment:
I definitely agree, however there are two reasons I went this route:

1. I thought it was less hacky than changing how we build the project, since the hack is limited to one file.
2. It is temporary, with a conrete, short term plan to get rid of it. (Once we add the login page.)